jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/multi/advance/AdvancedProcessDefDetailsMultiPresenter.java

```diff
@@ -56,27 +57,18 @@ public Position getPosition() {
     @WorkbenchMenu
     public Menus buildMenu() {
         return MenuFactory
-                .newTopLevelCustomMenu(newInstanceMenu).endMenu()
-
-                .newTopLevelCustomMenu(new MenuFactory.CustomMenuBuilder() {
-
-                    @Override
-                    public void push(MenuFactory.CustomMenuBuilder element) {
-                    }
-
-                    @Override
-                    public MenuItem build() {
-                        return new BaseMenuCustom<IsWidget>() {
-
-                            @Override
-                            public IsWidget build() {
-                                return view.getOptionsButton();
-                            }
-                        };
-                    }
-                }).endMenu()
-
                 .newTopLevelCustomMenu(new RefreshMenuBuilder(this)).endMenu()
+                .newTopLevelCustomMenu(newInstanceMenu).endMenu()
+                .newTopLevelMenu(Constants.INSTANCE.Options())
```

Comment:
Just out of curiosity, where does the magic happen which turns this into KebabMenu?

Reply:
that would be in here: https://github.com/kiegroup/appformer/pull/175/files#diff-30dc716694e59aff90189e2110212da5R163